### PR TITLE
[T4A4][T17-B1]Chia Zhe Min

### PR DESCRIPTION
**Pull request by zhems**

### CODE REVIEW

**Comment by zhems** [comment]
Ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -83,5 +83,17 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+   public String getPrintableString(Printable... printables){
+       String result = "";
+       for(Printable strings : printables) {
```
Comment (JoelT-92) [inline src/seedu/addressbook/data/person/Person.java:92]: Good use of foreach loop :+1:

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+import java.util.Objects;
+
+public interface Printable {
```
Comment (JoelT-92) [inline src/seedu/addressbook/data/person/Printable.java:5]: There should be a header comment for the interface

**Comment by JoelT-92** [comment]
@zhems Some comments added. Please close the PR after reading comments. Well done!

**Pull request closed without merging**